### DISCUSSION TITLE
How can Faster Whisper capture fast speech correctly?

### DISCUSSION THREAD

**Discussion by toanhuynhnguyen** [SYSTRAN/faster-whisper#1059] [Q&A]
I have an audio file, and a speaker speaks too fast. And Faster Whisper captures sentences incorrectly.
How can Faster Whisper capture fast speech correctly?
Thank you.

**Comment by techspecshq** [comment]
> I have an audio file, and a speaker speaks too fast. And Faster Whisper captures sentences incorrectly. How can Faster Whisper capture fast speech correctly? Thank you.

Pls upload the audio file
